src/config/development.tsx

```diff
@@ -51,9 +51,11 @@ const CELO_NETWORK_NUMBER = 44787;
 const CLASSIC_NETWORK_NUMBER = 63;
 const SOLANA_NETWORK: NonEVMChain = {
 	id: 0,
+	particleChainId: 103,
```

Comment:
I think the networkId will be better name since the backend also use this name, what do you think @RamRamez

Reply:
Done!